DEBUGGING.md

```diff
@@ -0,0 +1,101 @@
+## Debugging
+
+Debugging is one of the requested features of language support. Currently Kotlin native compiler produces debug info compatible
+with DWARF 2 specification, that allow modern debugger tools provide basic support:
+- breakpoints
+- stepping
+- type information
+
+### Producing binaries with debug info with kotlin compiler
+
+To produce binaries with kotlin compiler it's enough to use ``-g`` option in command line.<br/>
```

Comment:
To produce binaries with kotlin compiler it's enough to use ``-g`` option in command line. ->
To produce binaries with the _Kotlin/Native_ compiler it's sufficient to use ``-g`` option on the command line.